Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -164,7 +178,10 @@ bool ScrollAnchoringController::didFindPriorityCandidate(Document& document)
     if (RefPtr priorityCandidate = viablePriorityCandidateForElement(document.focusedElement())) {
         m_anchorElement = priorityCandidate;
         m_lastOffsetForAnchorElement = computeOffsetFromOwningScroller(*m_anchorElement->renderer());
-        LOG_WITH_STREAM(ScrollAnchoring, stream << "ScrollAnchoringController::viablePriorityCandidateForElement() for scroller: " << m_owningScrollableArea << " found priority candidate: " << *priorityCandidate);
+        if (auto element = elementForScrollableArea(m_owningScrollableArea))
```

Comment:
It's better to not have conditional code that only affects logging (we pay the cost for `elementForScrollableArea()` in release builds).

You can use `ValueOrNull` to log a pointer if necessary.

Reply:
Fixed.